doc/metadata.md

```diff
@@ -0,0 +1,64 @@
+
+*The DataSlice object containing extracted Data, is enriched with metadata computed from query execution. Below is the meta data list and a note explaining how they are calculated.*
+
+
+# H1 Where these metadata will be available ? 
+In order to provide access to these metada at each point where data are retrieved, a DataSlice object will be returned  instead of DataFrame.
```

Comment:
```suggestion
A DataSlice object will be returned  instead of DataFrame by each method that retrieve data, in order to provide access to these metadata everywhere. 
```